sbt_common/messaging/src/test/scala/uk/ac/wellcome/messaging/test/fixtures/Messaging.scala

```diff
@@ -206,22 +218,52 @@ trait Messaging
                        R]) = {
     withActorSystem { actorSystem =>
       withMockMetricSender { metricsSender =>
-        withLocalStackSqsQueue { queue =>
-          withExampleObjectMessageReaderFixtures {
-            case (bucket, messageReader) =>
-              withMessageWorker(
-                actorSystem,
-                metricsSender,
-                queue,
-                messageReader) { worker =>
+        withExampleObjectMessageReaderFixtures {
+          case (bucket, messageReader, queue) =>
+            withMessageWorker(actorSystem, metricsSender, queue, messageReader) {
+              worker =>
                 testWith((metricsSender, queue, bucket, worker))
-              }
-          }
+            }
         }
+
       }
     }
   }
 
+  def assertQueueEmpty(queue: Queue) = {
```

Comment:
Would these be more usefully defined on the SQS fixture, so they're available to other tests?

Reply:
yep, will move